doc/contributor/howto-guide-adding-generated-libraries.md

```diff
@@ -212,6 +213,5 @@ index c4ce00489..1858b48dc 100755
 
 ```shell
 git commit -m"Manually update READMEs, quickstart, and top-level stuff" \
-  "google/cloud/${library}" \
-  BUILD.bazel .codecov.yml .bazelignore
+  "google/cloud/${library}" BUILD.bazel
```

Comment:
At some point we mentioned how this omits any `ci/` edit produced during the previous "It is somewhat common" phase.  Perhaps we should address that here/now.

Reply:
Done.